src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -192,11 +218,19 @@ private void updateCaseInCcd(
             );
         } catch (FeignException exception) {
```

Comment:
We should also explicitly catch `FeignException.UnprocessableEntity` - in this case we should return a 200 response with errors and warnings from the called service. The caseworker needs to see those.